node/src/service.rs

```diff
@@ -472,6 +478,8 @@ where
 						slot_duration,
 						// We got around 500ms for proposing
 						block_proposal_slot_portion: SlotProportion::new(1f32 / 24f32),
+						// And a maximum of 750ms if slots are skipped
+						max_block_proposal_slot_portion: Some(SlotProportion::new(1f32 / 16f32)),
```

Comment:
Please mark the origin of the code (e.g. statemint).

Reply:
Added.